0000-deb-repo-additional-headers.md

```diff
@@ -169,30 +169,32 @@ of dependencies are involved, but still admin needs to be aware of it.
 The solution is simply add the following table:
 
 ```
-rhnPackageErrors
+rhnChannelError
 ================
-package_id  NUMERIC NOT NULL
+id          NUMERIC NOT NULL PRIMARY KEY
+channel_id  NUMERIC NOT NULL REFERENCES rhnChannel(id)
 error       VARCHAR NOT NULL
 ```
 
-During the synchronisation, reposync will log all errors about broken
-packages. Later on, UX can use this information for various purposes,
+During the synchronization, reposync will log all errors about broken
+packages. Later on, UI can use this information for various purposes,
 such as:
 1. Mark with an icon for the channel(s) has issues (permanent reminder)
 2. Display more detailed drill-down what packages are affected and why
 3. Display a warning/confirmation dialog for admin, once affected
    channel is attempted to be scheduled for something.
 
-In case package cannot be synchronised at all due to various reasons,
-the information about it still needs to be appearing in the database
-and marked as "broken". So then tools, like `spacewalk-data-fsck` can
-verify that the record is there, but the file isn't etc. As well as an
-error message can be placed to the log table "File was unable to be
-downloaded".
+In case package cannot be synchronized at all due to various reasons,
+the information about the failure needs to appear in the database.
+So then tools, like `spacewalk-data-fsck` can make use of this information.
 
 This information can be reused across all the channels, UI and CLI
 tools to display warning/status of a specified channel.
 
+The repo-sync process will not stop if a broken is encountered. An error will be inserted in the `rhnChannelError` table and the process will continue.
+No entry in `rhnPackage` will be created for broken packages (this is the current behavior) and no package will be saved to disk.
```

Comment:
Packages are saved on disk in a temporary area. I am not sure about the exact current status of the code, but they may stay there already until the next sync overwrite them.

Reply:
I've added [further steps in notes](https://github.com/uyuni-project/uyuni-rfc/pull/7/commits/183cb0e758445cf886fe1e0a30e0875b6b962656) about this.